inspirehep/modules/workflows/workflows/article.py

```diff
@@ -180,12 +181,20 @@
         is_arxiv_paper,
         [
             arxiv_fulltext_download,
+            arxiv_package_download,
```

Comment:
Note that merging and deploying to this is going to break the workflows that are stuck on the next task (there are 5 of them).